windows/web/slimevr_web_installer.nsi

```diff
@@ -394,9 +371,32 @@ Section "SlimeVR Server" SEC_SERVER
     WriteUninstaller "$INSTDIR\uninstall.exe"
 SectionEnd
 
-Section "SlimeVR Driver" SEC_VRDRIVER
+Section "Java JRE" SEC_JRE
     SectionIn RO
+    
+    Var /GLOBAL DownloadedJreFile
+    DetailPrint "Downloading Java JRE 17..."
+    ${If} ${RunningX64}
+        NScurl::http GET "https://github.com/adoptium/temurin17-binaries/releases/download/jdk-17.0.4.1%2B1/OpenJDK17U-jre_x64_windows_hotspot_17.0.4.1_1.zip" "$TEMP\OpenJDK17U-jre_x64_windows_hotspot_17.0.4.1_1.zip" /CANCEL /RESUME /END
+        StrCpy $DownloadedJreFile "OpenJDK17U-jre_x64_windows_hotspot_17.0.4.1_1"
+    ${Else}
+        NScurl::http GET "https://github.com/adoptium/temurin17-binaries/releases/download/jdk-17.0.4.1%2B1/OpenJDK17U-jre_x86-32_windows_hotspot_17.0.4.1_1.zip" "$TEMP\OpenJDK17U-jre_x86-32_windows_hotspot_17.0.4.1_1.zip" /CANCEL /RESUME /END
+        StrCpy $DownloadedJreFile "OpenJDK17U-jre_x86-32_windows_hotspot_17.0.4.1_1"
+    ${EndIf}
+    Pop $0 ; Status text ("OK" for success)
+    ${If} $0 != "OK"
+        Abort "Failed to download Java JRE 17. Reason: $0."
+    ${EndIf}
+    DetailPrint "Downloaded!"
+
+    DetailPrint "Unzipping Java JRE 17 to installation folder...."
+    nsisunz::Unzip "$TEMP\$DownloadedJreFile.zip" "$TEMP\$DownloadedJreFile\"
+    Pop $0
+    DetailPrint "Unzipping finished with $0."
+    CopyFiles /SILENT "$TEMP\$DownloadedJreFile\jdk-17.0.4.1+1-jre\*" "$INSTDIR\jre"
```

Comment:
Can this be matched without explicitly specifying the version or would that be unsafe?

Reply:
If you mean changing the `\jdk-17.0.4.1+1-jre\*` part to something like `\jdk-*\*`, then it looks like no. Getting `Copy failed` in this case.

Reply:
Understood, looks like this will just have to be manually updated each time, maybe a future refactor could try to install the jre in a less hardcoded way, but this is fine with the current state of it